pipelines/rj_crm__get_history_data/tasks.py

```
@@ -0,0 +1,375 @@
+        {body_inner}
+    </soapenv:Body>
+</soapenv:Envelope>"""
+    return envelope.encode("utf-8")
+
+
+def _parse_soap_response(xml_text: str) -> Tuple[str, str, List[Dict[str, str]]]:
+    """
+    Parseia a resposta SOAP de listagem de DataExtensions.
+    Retorna (overall_status, request_id, lista de DEs)
+    """
+    root = ET.fromstring(xml_text)
```

Ruff (0.15.20)

[error] Using `xml` to parse untrusted data is known to be vulnerable to XML attacks; use `defusedxml` equivalents

(S314)